hw/ip/aes/data/aes.hjson

```diff
@@ -22,7 +22,7 @@
   dv_doc:             "../doc/dv",
   hw_checklist:       "../doc/checklist",
   sw_checklist:       "/sw/device/lib/dif/dif_aes",
-  version:            "1.0.0",
+  version:            "1.0.1",
```

Comment:
Only increasing patch version because only the PRNG changed, which is not SW-visible.

CC @vogelpi